copper-server-impl/src/main/java/com/pogeyan/cmis/impl/services/CmisDiscoveryService.java

```diff
@@ -69,7 +72,12 @@ public static ObjectList getContentChanges(String repositoryId, Holder<String> c
 					.getObjectService(repositoryId, MTypeManagerDAO.class);
 			int maxItemsInt = maxItems == null ? 10 : maxItems.intValue();
 			if (changeLogToken == null || changeLogToken.getValue() == null) {
-				throw new CmisInvalidArgumentException("change log token value should not be null!");
+				TracingApiServiceFactory.getApiService().updateSpan(span,
+						new TracingMessages(String.format(ErrorMessages.TOKEN_VALUE_NULL, span.getTraceId()),
```

Comment:
Instead of creating "new TracingMessages", we can have a static function that acts as a factory.